src/mlpack/tests/ann/not_adapted/ann_layer_test.cpp

```diff
@@ -2739,7 +2739,7 @@ TEST_CASE("SimpleMultiplyMergeLayerTest", "[ANNLayerTest]")
 
 /**
  * Tests the LayerNorm layer.
- *
+ */
```

Comment:
Ah, actually to enable these tests, the tests should be removed from this file and then put in standalone files in `src/mlpack/tests/ann/layer/`.  If you want I can provide a patch to do that, just let me know. :+1:

Reply:
Sure, sounds good.

Reply:
I'm working on these.  Didn't realize they weren't actually compiling, so needed to fix them up.

Reply:
Let me know if I can be helpful for that; I'm not sure what state those tests were in.  (Once they compile... will they work??  We'll find out :smile:)

Reply:
Committed the tests for both LayerNorm and MultiheadAttention.